e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs

```diff
@@ -17,6 +17,7 @@ namespace IIoTPlatform_E2E_Tests.Standalone {
     /// </summary>
     [TestCaseOrderer("IIoTPlatform_E2E_Tests.TestExtensions.TestOrderer", TestConstants.TestAssemblyName)]
     [Collection("IIoT Platform Test Collection")]
+    [Trait("PublisherMode", "standalone")]
```

Comment:
can you add both name and values to the TestConstants?

Reply:
Done.